src/main/java/frc/robot/commands/CmdArmReset.java

```diff
@@ -21,14 +21,12 @@ public void execute()
     @Override
     public boolean isFinished() 
     {
-        if (_armSubsystem.isLimitSwitchPressed())
-        {
-            _armSubsystem.setExtensionMotorSpeed(0);
-            return true;
-        }
-        else
-        {
-            return false;
-        }
+        return _armSubsystem.isLimitSwitchPressed();
+    }
+
+    public void end(boolean interrupted)
```

Comment:
Since this function comes from `CommandBase`, we need to tell Java we're overriding the default behavior. Add `@Override` right above this function:

```suggestion
    @Override
    public void end(boolean interrupted)
```